shakenfist/client/main.py

```diff
@@ -602,6 +603,7 @@ def _show_instance(ctx, i, include_snapshots=False):
     print(format_string % ('cpus', i['cpus']))
     print(format_string % ('memory', i['memory']))
     print(format_string % ('disk spec', i['disk_spec']))
+    print(format_string % ('video', i['video']))
```

Comment:
video is a dict though, should we do a better job of formatting it here?

Reply:
The disk dumps a list of dict.
```
uuid        : c4d0bab2-6656-4a06-9f5f-0261c72e9f07
name        : bob
namespace   : system
cpus        : 1
memory      : 1024
disk spec   : [{'base': None, 'bus': None, 'size': 8, 'type': 'disk'}]
video       : {'memory': 16384, 'model': 'cirrus'}
node        : sf-3
power state : off
state       : created
console port: 48729
vdi port    : 38686

ssh key     : None
user data   : None

```
Change both or just video?
```
memory      : 1024
disk spec   : type:disk   bus:None  size:8   base:None
              type:disk   bus:ide   size:18  base:None
              type:cdrom  bus:ide   size:18  base:None
video       : model:cirrus  memory:16384
node        : sf-3
```
Defining characteristics of above:
* type is 5 char wide
* bus is min 4 wide
* size is min 2 char wide
* no space after colon
* two spaces after value

To make this more complicated I like moving the colon but I know everybody has a different taste.
```
uuid:          c4d0bab2-6656-4a06-9f5f-0261c72e9f07
name:          bob
namespace:     system
cpus:          1
memory:        1024
disk spec:     type:disk   bus:None  size:8   base:None
               type:disk   bus:ide   size:18  base:None
               type:cdrom  bus:ide   size:18  base:None
video:         model:cirrus  memory:16384
node:          sf-3
power state:   off
state:         created
console port:  48729
vdi port:      38686

ssh key:       None
user data:     None
```

Reply:
I think this proposed format is much nicer that the current dumped JSON. I think we should change both. I think either format is fine as long as its neat, so you're choice.